test/Fixtures.purs

```diff
@@ -743,6 +853,10 @@ txFixture7 =
         , networkId: Nothing
         , collateralReturn: Nothing
         , totalCollateral: Nothing
+        , votingProposals: []
+        , votingProcedures: VotingProcedures.empty
```

Comment:
I'm quite worried about the need to import this type just to define a tx body. can we make this type a right-biased semigroup with `Map.unionWith (Map.unionWith ...)` append operation and make it also a monoid?

Reply:
We can, but can you please clarify what the issue is with using `empty`?

Reply:
Addressed in https://github.com/mlabs-haskell/purescript-cardano-types/pull/6/commits/af365a1e4ae7ef0f6f0f91e080e41d95380af20e

Reply:
Thank you!